packages/cms-lib/http.js

```diff
@@ -106,8 +106,15 @@ const createGetRequestStream = ({ contentType }) => async (
   options,
   destination
 ) => {
-  const { query, ...rest } = options;
-  const requestOptions = addQueryParams(rest, query);
+  const { query, uri, ...rest } = options;
+  const requestOptions = addQueryParams(
+    {
+      ...rest,
+      // Handle extended characters(kanji, etc)
+      uri: encodeURI(uri),
```

Comment:
I think that this is okay, although I wondered if `encodeURIComponent()` in `fetchFileStream` was the way to go like we do in the `upload` api helper.

Reply:
I did that initially but the requests all fail when using `encodeURIComponent` in place of `encodeURI`.

Reply:
if `uri` is a complete URI then we want to use `encodeURI`.  `encodeURIComponent` is for paths, queries, etc.

```javascript
encodeURIComponent('https://foo.com?x=a b') == "https%3A%2F%2Ffoo.com%3Fx%3Da%20b"

encodeURI('https://foo.com?x=a b') == "https://foo.com?x=a%20b"
```

Reply:
@corymartin yeah, i was suggesting we do it only to the path like we do in the upload api helper via something like `${FILE_MAPPER_API_PATH}/download/${encodeURIComponent(filepath)}`. I actually think we have to do in the hlpers to avoid double encoding.